integreat_cms/cms/models/users/user_fido_key.py

```diff
@@ -5,9 +5,9 @@
 from ..abstract_base_model import AbstractBaseModel
 
 
-class UserMfaKey(AbstractBaseModel):
+class UserFidoKey(AbstractBaseModel):
     """
-    Data model representing a user's multi-factor-authentication (MFA) key
+    Data model representing a user's FIDO key
     """
 
     user = models.ForeignKey(
```

Comment:
I am not really sure about that as there is still some development ongoing with FIDO or if we might start supporting another standard. If so, we might add an type attribute to the model and keep the model and objects as they are. But if you prefer to be clear about the current keys, I would be fine with renaming them :man_shrugging:

Reply:
Ok, but since the model already is called `UserFidoKey`, it will probably be renamed when we add more key types? If we want to stay flexible, shouldn't we just keep it `UserMfaKey`?
I mean the key model fields might also differ for different key types, so I think it would make sense to create an own model for each key type...

Reply:
Okay, I'll streamline it accordingly :+1: